### PR TITLE
Added missing include file for release notes

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Added missing include file and fixed liquid link formatting

### CODE REVIEW

**Comment by lorikrell** [comment]
This PR includes the actual 2.3.2 new release notes files. Are these also up for full review? Just checking.

File: extensions/amazon-sales/index.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Amazon Sales Channel install is a `.zip` file available from the Magento Marketp
       composer require magento/services-connector:~1.0.3
       ```
 
-    - Enter your [authentication keys]({{https://devdocs.magento.com/guides/v2.3/install-gde/prereq/connect-auth.html}}). Your public key is your username; your private key is your password.
+    - Enter your [authentication keys]({{ site.baseurl }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```
Comment (meker12) [inline extensions/amazon-sales/index.md]:
```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```

Comment (dshevtsov) [reply extensions/amazon-sales/index.md]: this should be `{{ page.baseurl }}`

File: extensions/amazon-sales/index.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ Amazon Sales Channel install is a `.zip` file available from the Magento Marketp
       composer require magento/services-connector:~1.0.3
       ```
 
-    - Enter your [authentication keys]({{https://devdocs.magento.com/guides/v2.3/install-gde/prereq/connect-auth.html}}). Your public key is your username; your private key is your password.
+    - Enter your [authentication keys]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```
Comment (meker12) [inline extensions/amazon-sales/index.md]:
```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```

File: extensions/amazon-sales/index.md [hunk 3]
```diff
@@ -31,9 +31,9 @@ Amazon Sales Channel install is a `.zip` file available from the Magento Marketp
       composer require magento/services-connector:~1.0.3
       ```
 
-    - Enter your [authentication keys]({{https://devdocs.magento.com/guides/v2.3/install-gde/prereq/connect-auth.html}}). Your public key is your username; your private key is your password.
+    - Enter your [authentication keys]({{ site.baseurl }}/guides/{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```
Comment (meker12) [inline extensions/amazon-sales/index.md]:
```suggestion
    - Enter your [authentication keys]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
```

**Pull request merged**